docs/src/main/tut/pull_request.md

```diff
@@ -128,6 +130,57 @@ createPullRequestIssue.exec[cats.Id, HttpResponse[String]]() match {
 
 See [the API doc](https://developer.github.com/v3/pulls/#create-a-pull-request) for full reference.
 
+## List pull requests
+
+You can list the reviews for a pull request using `listReviews`, it takes as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository).
+- the pull request id.
+
+As an example, if we wanted to see all the reviews for pull request 139 of `47deg/github4s`:
+
+```scala
```

Comment:
You can use `tut:silent` instead of `scala` for code examples that do not create stuff.
That way your examples will be compiled when you run `makeMicrosite`.